hoaxshell.py

```diff
@@ -301,22 +301,25 @@ def do_POST(self):
 			content_len = int(self.headers.get('Content-Length'))
 			output = self.rfile.read(content_len)
 			
-			try:
-				bin_output = output.decode('utf-8').split(' ')
-				to_b_numbers = [ int(n) for n in bin_output ]
-				b_array = bytearray(to_b_numbers)
-				output = b_array.decode('utf-8', 'ignore')
-				
-			except UnicodeDecodeError:
-				print(f'[{WARN}] Decoding data to UTF-8 failed. Printing raw data.')
-			
-			if isinstance(output, bytes):
-				pass
-				
+			if output:
+				try:
+					bin_output = output.decode('utf-8').split(' ')
+					to_b_numbers = [ int(n) for n in bin_output ]
+					b_array = bytearray(to_b_numbers)
+					output = b_array.decode('utf-8', 'ignore')
+
+				except UnicodeDecodeError:
+					print(f'[{WARN}] Decoding data to UTF-8 failed. Printing raw data.')
+
+				if isinstance(output, bytes):
+					pass
+
+				else:
+					output = output.strip() + '\n' if output.strip() != '' else output.strip()
+					
+				print(f'\r{GREEN}{output}{END}')
 			else:
-				output = output.strip() + '\n' if output.strip() != '' else output.strip()
-				
-			print(f'\r{GREEN}{output}{END}') if output not in [None, ''] else print(f'\r{ORANGE}No output.{END}')
+				print(f'\r{ORANGE}No output.{END}')
```

Comment:
if you change this for a pass instead of 'No output.' to stdout the experience is smoother-  
<img width="913" alt="image" src="https://user-images.githubusercontent.com/14915771/180607781-075c8c27-731d-4b74-b878-1050605c6bb3.png">

Reply:
line 322 - commented out the part I'm referring to for reference - 
<img width="1047" alt="image" src="https://user-images.githubusercontent.com/14915771/180607821-7c63ca76-e3cb-425f-9178-20773737a453.png">

Reply:
I would agree! Actually the whole `else` part can be deleted